plexapi/__init__.py

```diff
@@ -17,6 +17,7 @@
 VERSION = '3.0.6'
 TIMEOUT = CONFIG.get('plexapi.timeout', 30, int)
 X_PLEX_CONTAINER_SIZE = CONFIG.get('plexapi.container_size', 100, int)
+X_PLEX_ENABLE_FAST_CONNECT = CONFIG.get('plexapi.enable_fast_connect', False, bool)
```

Comment:
Any downside to let this just be enabled by default?

Reply:
There is a chance that you'll connect to an external IP while you're in your home network. It may be a bit unexpected and strange.